packages/laconia-core/test/laconia.spec.js

```diff
@@ -206,30 +201,34 @@ describe("laconia", () => {
 
   describe("callback behaviour", () => {
```

Comment:
Most of the tests within this describe block doesn't seem to be related to callback anymore, except the "should not call Lambda callback. Should we pull them out?

Reply:
Agreed.